better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
@matheu I advised to just use __DIR__ in this case, the plugin was already using 5.3+ syntax. see ```get_instance``` method on this class.